site/docs/connectors/index.md

```diff
@@ -145,6 +147,15 @@ The table below lists the repository connectors that act as an adapter for third
     - [Setting up a governance engine content pack](/guides/developer/open-metadata-archive/creating-governance-engine-content-packs) to create a [governance action engine](/concepts/governance-action-engine) definition to load into a [Metadata Access Store](/concepts/metadata-access-store).
     - [Writing a governance action service](/guides/developer/governance-action-services/overview) for information on writing new governance action services.
 
+### Governance Daemon Connectors
+
+---8<-- "docs/connectors/governance-daemon/governance-daemon-intro.md"
+
+| Connector | Description |
+|---|---|
+| [Open Lineage Janus Connector](/connectors/runtime/open-lineage-janus-connector) | The Open Lineage connectors provide plugins to the [Open Lineage Server](/services/open-lineage-services) that allow the Open Lineage Services to connect with databases.
```

Comment:
isn't the correct link /connectors/governance-daemon/open-lineage-janus-connector (?)